test/scripts/performance/mt02_verify_tenants_creation_performance.bash

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+
+# The script could be used for Multitenants Performance verification
+# script could be used together with multitenancy_performance.go, to provide flexibility for testing
+# Notes:
+#   IDP should be created before script execution.
+#       NUM_REGULAR_USER in scripts/setup-sso-idp.sh need to be changed before running IDP creation
+#   Promotions options in the script are not checked yet and could require updating. Use PROMOTE_STAGE=false and PROMOTE_PROD=false
+#   Recommended to use multitenancy_performance.go for Cluster users creation, as usersLogin function is not creating Cluster users_
+# Script require improvement, but could be used for performance investigation
+
+ACCESS_TOKEN="xxxxxxx"
+ADMIN_URL="https://3scale-admin.apps.xxxx.xx.s1.devshift.org"
+CLUSTER_WIDLCARD_URL="xxxxx.xxx.s1.devshift.org"
+INSTALL_TYPE_PREFIX="sandbox"
+USER_BASENAME="test-user"
+USER_PASSWD="Password1"
+ADMIN_API_TOKEN="sha256~#####"
+MIN=1
+MAX=200
+CREATE_CRS=true
+CREATE_PROD=true
+PROMOTE_STAGE=false
+PROMOTE_PROD=false
+DO_USERS_LOGIN=false
+
+createTenantCR() {
+  cat <<EOT >> Tenant.yaml
+  apiVersion: integreatly.org/v1alpha1
+  kind: APIManagementTenant
+  metadata:
+    name: example
+    namespace: "$USER_BASENAME$i-dev"
+  spec: {}
+EOT
+  oc create -f Tenant.yaml
+  rm Tenant.yaml
```

Comment:
I realise this script is somewhat unfinished but minor suggestion to save re-creating and removing the Tenant.yaml every iteration:

```suggestion
  cat << EOF | oc create -f -
apiVersion: integreatly.org/v1alpha1
kind: APIManagementTenant
metadata:
  name: example
  namespace: "$USER_BASENAME$i-dev"
spec: {}
EOF
```

Reply:
Done. Thank you for comment.